api/CredentialsContainer.json

```
@@ -640,6 +640,55 @@
                   "deprecated": false
                 }
               }
+            },
+            "prf": {
+              "__compat": {
+                "description": "`prf` extension (pseudo-random function)",
+                "spec_url": "https://w3c.github.io/webauthn/#prf-extension",
+                "tags": [
+                  "web-features:webauthn"
+                ],
+                "support": {
+                  "chrome": {
+                    "version_added": false
```

Based on https://chromiumdash.appspot.com/commit/9b8e20e8c56b9e193d7e961e546e2c44b21094a1, Chrome 116 supports it.

```suggestion
                    "version_added": "116"
```